flang/test/Fir/CUDA/cuda-extranal-mangling.mlir

```diff
@@ -1,13 +1,25 @@
 // RUN: fir-opt --split-input-file --external-name-interop %s | FileCheck %s
 
+module @mod attributes {gpu.container_module} {
+
 gpu.module @cuda_device_mod {
-  gpu.func @_QPfoo() {
+  gpu.func @_QPfoo() kernel {
```

Comment:
Is there a type in the filename ?

Reply:
Yes! Good catch! It was from a previous commit. I'll address that